NightscoutServiceKit/Extensions/StoredDosingDecision.swift

```diff
@@ -35,12 +35,19 @@ extension StoredDosingDecision {
     }
     
     var recommendTempBasal: RecommendedTempBasal? {
-        guard let tempBasalRecommendationDate = tempBasalRecommendationDate else {
+        guard let recommendedTempBasal = recommendedTempBasal else {
             return nil
         }
-        return RecommendedTempBasal(timestamp: tempBasalRecommendationDate.date,
-                                    rate: tempBasalRecommendationDate.recommendation.unitsPerHour,
-                                    duration: tempBasalRecommendationDate.recommendation.duration)
+        return RecommendedTempBasal(timestamp: recommendedTempBasal.date,
+                                    rate: recommendedTempBasal.recommendation.unitsPerHour,
+                                    duration: recommendedTempBasal.recommendation.duration)
+    }
+
+    var recommendBolus: Double? {
```

Comment:
[nit] Similarly here: `recommendBolus` and `recommendedBolus` are really hard to tell apart.

Reply:
I renamed everything so it is clear.